src/Test.Utilities.Sarif/TestConstants.cs

```diff
@@ -1,6 +1,9 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Microsoft.CodeAnalysis.Sarif;
+using System;
```

Comment:
>System [](start = 6, length = 6)

these namespaces are out of order. fix your VS settings to put System.* directives first.

Reply:
Didn't even know there was a setting for that, but yes, that is also my preferred order.

---
In reply to: [390639230](https://github.com/microsoft/sarif-sdk/pull/1808#discussion_r390639230) [](ancestors = 390639230)